### PR TITLE
fix: minimum locking period is not 1 week

### PR DESCRIPTION
**Pull request by sander2**

### CODE REVIEW

File: docs/kintsugi/governance.md [hunk 1]
```diff
@@ -62,12 +62,14 @@ To vote on governance proposals, users must lock KINT with the Kintsugi parachai
 
 - The longer KINT are locked, the more vKINT are minted.
 - Lock periods:
-  - **Minimum: 1 week**
```
Comment (gregdhill) [inline docs/kintsugi/governance.md]: I think this would also be technically correct, but happy to change this too.

Comment (nud3l) [inline docs/kintsugi/governance.md]: can you please keep the minimum? Just to make it clear that this is the minim period

File: docs/kintsugi/governance.md [hunk 2]
```diff
@@ -62,12 +62,14 @@ To vote on governance proposals, users must lock KINT with the Kintsugi parachai
...
+  - **Granularity: 1 week**
   - **Maximum: 96 weeks** (2x the maximum parachain lease period on Kusama).
 - As time progresses, the vKINT balance **decreases linearly with time** (per block)
 - KINT can be withdrawn at the end of the lock period (all at once)
 - You can extend the lock period at any time (up to the maximum lock period)
 
+?> For performance reasons, the end dates of locks are rounded down to the nearest multiple of the granularity.
```
Comment (nud3l) [inline docs/kintsugi/governance.md]:
```suggestion
?> For performance reasons, the end dates of locks are rounded down to the nearest multiple of the granularity, i.e., KINT can be locked in weekly intervals.
```

File: docs/kintsugi/governance.md (hunk 1 above)
Comment (sander2) [reply docs/kintsugi/governance.md]: That's the thing, it is not technically the minimum. Every week there is a checkpoint. If you create_lock, say, 2 days before such a checkpoint, you can reserve for 2 days.

Comment (gregdhill) [reply docs/kintsugi/governance.md]: @sander2 create locks rounds it down though right?

Comment (sander2) [reply docs/kintsugi/governance.md]: It rounds down to the checkpoint, yes. If you're 2 days from a checkpoint and you try to reserve for 3 days, you will actually lock it for 2

Comment (gregdhill) [reply docs/kintsugi/governance.md]: I don't think that's correct, you wouldn't be able to lock it for 2 days. Assuming the minimum is a week and you try to lock for less than 7 days the following calculation would return 0: https://github.com/interlay/interbtc/blob/b4c86e64637abe7f711c527ae16d6749be65ea9a/crates/escrow/src/lib.rs#L356

Comment (sander2) [reply docs/kintsugi/governance.md]: No it wouldn't - the argument to this function is not the length, but the end height

**Pull request merged**